components/automate-ui/src/app/helpers/auth/sorter.ts

```diff
@@ -0,0 +1,22 @@
+
+export class ChefSorters {
```

Comment:
[nit] extra line above.

does this helper need to live under `auth`? doesn't seem specific to auth, except that our pages are using it the most right now.